jina/serve/runtimes/request_handlers/data_request_handler.py

```diff
@@ -47,6 +47,7 @@ def _load_executor(self):
                     'py_modules': self.args.py_modules,
                 },
                 extra_search_paths=self.args.extra_search_paths,
+                kwargs={'runtime_args': {}},
```

Comment:
do we still need this ?

Reply:
Yes, otherwise there are some errors in the parsing :thinking: it is a bit hacky but I don't think that it is worth the effort to investigate what is going on